opensafely/__init__.py

```diff
@@ -57,6 +57,11 @@ def show_help(**kwargs):
     parser_check.set_defaults(function=check.main)
     check.add_arguments(parser_check)
 
+    # Add `notebook` subcommand
+    notebook_check = subparsers.add_parser("notebook", help=notebook.DESCRIPTION)
+    notebook_check.set_defaults(function=notebook.main)
```

Comment:
I think you want `parser_notebook` here if you're going for consistency with the existing code.

Reply:
oh, c/p error yes